framework/source/class/qx/ui/treevirtual/SimpleTreeDataModel.js

```diff
@@ -389,8 +389,8 @@ qx.Class.define("qx.ui.treevirtual.SimpleTreeDataModel",
           return;
         }
         node.columnData[columnIndex] = value;
+        this._rowArr[rowIndex][columnIndex] = value;
       }
-      this.setData();
```

Comment:
So I think this change is valid except for one thing. The call to `setData` would also cause this code to be run:

```
        if (columnIndex == this.__sortColumnIndex) {
          this.clearSorting();
        }
```
which generates a `metaDataChanged` event which causes all content to be updated. I believe that that includes sorting. So with this change, it appears that if the column that the data being changed is in is the sorted column of the table, the table won't be resorted. Can you confirm?

Reply:
I can't see how `setData()` would cause that code to run - the code you've highlighted is in the `setValue` of `qx.ui.table.model.Simple` I think, but we are overriding with the `setValue` in `qx.ui.treevirtual.SimpleTreeDataModel`?
Also, probably sorting of columns should not be allowed in a TreeVirtual with columns, the tree would not look very pretty 😄

Reply:
Doh! Yeah, of course you're right about sorting a tree based on other columns.

The way it does what I said was, as I traced through it, because `setData` via `clearSorting` generated that `metaDataChanged` which called into Scroller and then each Pane. 

But it's all moot since sorting a tree is bogus. I *think* this change is safe. It's risky, in that it can affect a great number of applications. Since the release isn't imminent, I think we can probably go with it, and see if anyone encounters problems. I'm a bit scared, though...

Reply:
I still don't see it. The `setData` method is also overridden by the SimpleTreeDataModel wherein it doesn't use the `clearSorting` flag.  I'm happy to wait a while to merge this whilst I perform more testing. We have heavy use of TreeVirtualTable so hopefully would pick up any problems.